packages/tabs-overflow/src/spectrum-config.js

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
This pattern does not currently have a corollary in Spectrum CSS, so we should remove this file so all of the tooling can run appropriately...